### ISSUE TITLE
Log in with github button does not work

### ISSUE THREAD

**Issue by lilincmu** [binarylu/leetcode-ext#9]
When first clicking the button, a login page will pop up. But after type in the github account info, nothing happens. After that, even clicking the "Log in with github" button multiple times, no page will pop up. I'm using v1.1.4  and everything else works fine except for github extension.

**Comment by binarylu** [comment]
If I didn't unpublish this extension, you can reinstall to fix. I'll publish recently, you can wait next version or you can inspect the extension to clean the broken data and try to login again. I'll publish ASAP.

**Comment by lilincmu** [comment]
OK! Just want to make sure it's not only my problem. I'll wait until the next version. Thank you!

**Comment by zhufuzz** [comment]
This is the most amazing plugin I have ever had. This is the first time I think the plugin worth more than my laptop. Can you provide a paypal account or something, so I can donate some money? Seriously.

**Comment by zhaodelong** [comment]
Yes. I have the same problem. After logged in, I cannot add repo for leetcode practice. The next time I open the extension. There is still a log in button. I added my own client id, client secret id and added it to chrome in developer mode.

**Comment by snaveenp** [comment]
I am also facing the same issue. :(
